release/release.go

```diff
@@ -37,41 +37,24 @@ type OS struct {
 	VersionID string `json:"version-id,omitempty"`
 }
 
+var (
+	apparmorFeaturesSysPath  = "/sys/kernel/security/apparmor/features"
+	requiredApparmorFeatures = []string{"dbus", "mount", "namespaces", "networking", "ptrace", "signal"}
```

Comment:
It is 'network', not 'networking'. You should also look for 'domain', 'file' and 'caps'